README.md

```diff
@@ -65,126 +82,49 @@ To run an interactive yocto shell for `cpu01-edgefarm`
 ```bash
-export MENDER_DEVICE_ID=<device id from mender portal>
+MENDER_DEVICE_ID=<device id from mender portal>
 ```
+
 Start deployment
+
 ```bash
 ./dobi.sh cpu01-edgefarm-mender-deploy
 ```
 
-Call `./dobi.sh list` to see a list of all jobs. 
-
-#### Yocto download and shared state cache
-
-By default, Yocto downloads will be placed into the *./downloads* folder, in order
-to share the downloads between all projects.
-
-By default, Yocto shared state cache will be placed into *\yocto/sstate-cache* folder.
-
-You can set default environment variables in the *default.env* file (template is provided: *default.env.template*), for example:
-
-**Note:**
-
-**YOCTO_DOWNLOAD_DIR must be specified without trailing "/", while
-YOCTO_SSTATE_CACHE_DIR must be specified *with* trailing "/"**
-
-```
-# Version of dobi to download, if not in $PATH
-DOWNLOAD_VERSION_DOBI="0.13.0"
-
-# Set the default docker namespace
-DOCKER_NAMESPACE=harbor.ci4rail.com
-
-# Directory for Yocto downloads.
-# If left empty: downloads go into project local directory
-# otherwise: Specify absolute path without trailing "/"
-YOCTO_DOWNLOAD_DIR=/opt/yocto/downloads/
-
-# Directory for Yocto shared state cache.
-# If left empty: state cache goes into project local directory
-YOCTO_SSTATE_CACHE_DIR=/opt/yocto/sstate-cache/
-
-# Mender artifact development suffix
-MENDER_ARTIFACT_DEV_SUFFIX=-dev
-
-# Yocto image name suffix
-IMAGE_NAME_SUFFIX=-dev
-```
-
-## CI/CD 
-
-The following provides an overview of the different dobi and ci pipelines together with their configuration files:
-
-Different CI pipelines exist:
-
-* triger when a new commit is made on the main branch of this repo:
-    * Testing stage. Triggers on any commit in main banch
-        * cpu01-edgefarm: Builds the image and runs acceptance test in testfarm
-        * cpu01-bringup: Builds the image 
-        * cpu01-bringup: Builds the image 
-    * Staging stage. Triggers on commits with "staging" tag (not yet implemented)
-    * Production stage. Triggers on commits with "staging" tag (not yet implemented)
-
-* trigger on github pull requests:
-    * cpu01-edgefarm: Builds the image and runs acceptance test in testfarm
-    * Other images are not built and tested during pull requests
-
-![Yocto pipelines](/doc/yocto_pipelines.svg)
+### Deploy images via minio
 
-### General perparations
-
-First download `fly` from your concourse server. The production concourse server that is reachable at https://concourse.ci4rail.com.
+Upload image to minio, e.g.:
 
 ```bash
-sudo wget https://concourse.ci4rail.com/api/v1/cli?arch=amd64&platform=linux -O /usr/local/bin/fly
-sudo chmod +x /usr/local/bin/
+./dobi.sh cpu01-edgefarm-minio-push
 ```
 
-Then login to your concourse instance.
+To deploy an image to a specific device set the ip address of raspberry pi connected to this device, if you haven't specified it in `config/custom.env`:
```

Comment:
avoid "raspberry pi" -> use a generic name, in case we change one day to another test computer

Reply:
ok